src/me/john000708/barrels/Barrel.java

```diff
@@ -24,20 +25,20 @@
 import org.bukkit.inventory.Inventory;
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.material.MaterialData;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 /**
  * Created by John on 06.05.2016.
  */
 public class Barrel extends SlimefunItem {
 
-	private static final String LORE_DATA = ChatColor.translateAlternateColorCodes('&', "&b&a&r&r&e&l");
-	
-    private static final int[] border1 = {0, 1, 2, 9, 11, 18, 19, 20};
-    private static final int[] border2 = {3, 5, 12, 13, 14, 21, 23};
-    private static final int[] border3 = {6, 7, 8, 15, 17, 24, 25, 26};
+    private int[] border1 = {0, 1, 2, 9, 11, 18, 19, 20};
+    private int[] border2 = {3, 5, 12, 13, 14, 21, 23};
+    private int[] border3 = {6, 7, 8, 15, 17, 24, 25, 26};
```

Comment:
These should be static and final, otherwise you are just allocating unuseful RAM.